datamol/reactions/_reactions.py

```diff
@@ -197,6 +202,7 @@ def select_reaction_output(
 def apply_reaction(
     rxn: dm.ChemicalReaction,
     reactants: tuple,
+    product_index: Union[int, list] = 0,
```

Comment:
The signature is difference from the one in `select_reaction_output`. They should  be same right?

Reply:
Yes. They are the same.